src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -48,7 +48,7 @@ public void Canvas_Float(ExpressionTestCase testCase)
                 ConsistentOneColumnTableResult = true
             };
 
-            RunExpressionTestCase(testCase, features, numberIsFloat: true);
+            RunExpressionTestCase(testCase, features, numberIsFloat: true, Summary);

```

Comment:
>Summary [](http://example.com/codeflow?start=75&length=7)

what is this flag?

Reply:
should probably be renamed
it's ITestOutputHelper == xUnit logger